### PR TITLE
Added text output to file

### PR DESCRIPTION
**Pull request by Shurron**
Completed issue #16

### CODE REVIEW

File: src/QuizData.TextReport/TextReporter.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+﻿using System.IO;
+using QuizData.Analyser.Models;
+
+namespace QuizData.TextReport
+{
+	public class TextReporter
+	{
+		public static void ToFile(string path, DataAnalyserReport report)
```
Comment (vkamiansky) [inline src/QuizData.TextReport/TextReporter.cs]: Add ability to test this via streams.

**Review by vkamiansky — CHANGES_REQUESTED** [review]
Requires a unit test project.

**Review by vkamiansky — APPROVED** [review]

**Review by vkamiansky — CHANGES_REQUESTED** [review]
Add test for reporter.

**Comment by Shurron** [comment]
> Add test for reporter.

Done in [Added one unit test for TextReporter](https://github.com/vkamiansky/analysis-practice-jul-2019/commit/7476a87b65d3de28d53d9c6222df6a094a946e2a)

**Review by vkamiansky — APPROVED** [review]

**Pull request merged**